File: android/src/main/kotlin/io/axept/axeptio_sdk/AxeptioEventStreamHandler.kt
package io.axept.axeptio_sdk

import io.flutter.plugin.common.EventChannel

/** No-op stream handler — events are emitted by the Flutter WebView layer. */
object AxeptioEventStreamHandler : EventChannel.StreamHandler {
    override fun onListen(arguments: Any?, events: EventChannel.EventSink?) {}
    override fun onCancel(arguments: Any?) {}
}

File: ios/Classes/AxeptioEventStreamHandler.swift
import Flutter

/// No-op stream handler — events are emitted by the Flutter WebView layer.
class AxeptioEventStreamHandler: NSObject, FlutterStreamHandler {
  func onListen(withArguments arguments: Any?, eventSink events: @escaping FlutterEventSink)
    -> FlutterError?
  {
    return nil
  }

  func onCancel(withArguments arguments: Any?) -> FlutterError? {
    return nil
  }
}
